extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -17,14 +34,26 @@ public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $ou
 		$wg = F::app()->wg;
 		$title = $wg->Title->getPrefixedDBkey();
 
-		if ( isset( $wg->articleVideoFeaturedVideos[$title]['videoId'] ) ) {
+		if ( $wg->enableArticleFeaturedVideo && isset( $wg->articleVideoFeaturedVideos[$title]['videoId'] ) ) {
 			$vars['wgOoyalaParams'] = [
 				'ooyalaPCode' => $wg->ooyalaApiConfig['pcode'],
 				'ooyalaPlayerBrandingId' => $wg->ooyalaApiConfig['playerBrandingId'],
 			];
 			$vars['wgFeaturedVideoId'] = $wg->articleVideoFeaturedVideos[$title]['videoId'];
 		}
 
+		if ( $wg->enableArticleRelatedVideo && isset( $wg->articleVideoRelatedVideos ) ) {
+			$relatedVideo = ArticleVideoController::getRelatedVideoData( $wg->articleVideoRelatedVideos, $title );
+
+			if ( isset( $relatedVideo['videoId'] ) ) {
+				$vars['wgOoyalaParams'] = [
```

Comment:
If you extract the checks as described in https://github.com/Wikia/app/pull/12529/files#r104894328 then we could set it only once `if ( shoudLoadArticleFeaturedVideo() || shouldLoadArticleRelatedVideo() )`